tests/functional/aws-node-sdk/test/bucket/getBucketNotification.js

```diff
@@ -0,0 +1,80 @@
+const assert = require('assert');
+const { errors } = require('arsenal');
+const { S3 } = require('aws-sdk');
+
+const getConfig = require('../support/config');
+const BucketUtility = require('../../lib/utility/bucket-util');
+
+const bucket = 'notificationtestbucket';
+const notificationConfig = {
+    QueueConfigurations: [{
+        Events: ['s3:ObjectCreated:*'],
+        QueueArn: 'arn:scality:bucketnotif:::target1',
+        Id: 'test-id',
+    }],
+};
+
+// Check for the expected error response code and status code.
+function assertError(err, expectedErr, cb) {
+    if (expectedErr === null) {
+        assert.strictEqual(err, null, `expected no error but got '${err}'`);
+    } else {
+        assert.strictEqual(err.code, expectedErr, 'incorrect error response ' +
+            `code: should be '${expectedErr}' but got '${err.code}'`);
+        assert.strictEqual(err.statusCode, errors[expectedErr].code,
+            'incorrect error status code: should be 400 but got ' +
+            `'${err.statusCode}'`);
```

Comment:
Not sure I would add extra messages here as the asserted error will already output something very similar to this.